kafka-ui-react-app/src/components/Topics/Topic/SendMessage/SendMessage.tsx

```diff
@@ -106,8 +106,8 @@ const SendMessage: React.FC = () => {
           clusterName,
           topicName,
           createTopicMessage: {
-            key: !key ? null : key,
-            content: !content ? null : content,
+            key: !key ? undefined : key,
```

Comment:
@NelyDavtyan this change should not occur. Please regen your types locally first `npm run gen:sources`
ping: @Haarolean

Reply:
@workshur, thanks.